src/main/java/com/fakng/fakngagrgtr/vacancy/VacancyDto.java

```diff
@@ -14,5 +15,5 @@ public class VacancyDto {
     private String url;
     private LocalDateTime addDate;
     private CompanyDto company;
-    private LocationDto location;
+    private List<LocationDto> location;
```

Comment:
Don't use LocationDto from the google aspect. Differ dto from Google API response and dto from REST API.

Reply:
It's another one, I guess. Just the same naming